syncd/MdioIpcServer.h

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <ctype.h>
+#include <errno.h>
+#include <string.h>
+#include <sys/types.h>
+#include <sys/select.h>
+#include <sys/socket.h>
+#include <sys/un.h>
+#include <time.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <pthread.h>
+
+extern "C" {
+#include <sai.h>
+#include <saiversion.h>
+}
+
+namespace syncd
+{
+    class MdioIpcServer
+    {
+        public:
+
+            MdioIpcServer(int globalContext);
```

Comment:
please look like other classes handle single paerameters

Reply:
Code is changed as suggested.